course_content/notebooks/dask_intro.ipynb

```diff
@@ -4,9 +4,45 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# An Introduction to Dask\n",
+    "# An Introduction to Dask"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Course overview\n",
+    "\n",
+    "Dask is a flexible parallel computing library for Python. It allows dynamic scheduling of tasks for optimising computations and includes collections (extensions of arrays, lists and dataframes) for use with parallel data processing.\n",
+    "\n",
+    "As of Iris v2.0, dask has been integrated with Iris. This means you can utilise all of the benefits of dask while keeping the fileformat interoperability and other functionality that Iris provides."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Why does this course exist?\n",
+    "\n",
+    "This course exists to enable you to use dask and Iris together for more performant, memory efficient data processing by showing useful patterns for combining dask and Iris.\n",
+    "\n",
+    "#### Iris/dask integration\n",
+    "\n",
+    "Iris uses dask to defer data loading and processing by default in a number of situations, including loading and some mathematical, analytical and statistical operations. This course will introduce patterns for producing your own bespoke data processing pipelines that leverages Iris and dask functionality \n",
+    "\n",
+    "This course will introduce some patterns for using Iris and dask together to produce your own bespoke data prcessing pipelines that meet your analytical needs while maintaining deferred data.\n",
+    "\n",
+    "\n",
+    "#### Performance\n",
+    "\n",
+    "A real benefit of dask is parallel data processing by default. This means you will see existing Iris processing paradigms getting faster, and we will show patterns for loading and processing data to take advantage of dask parallel processing so that your bespoke processing pipelines will also run faster.\n",
+    "\n",
+    "\n",
+    "#### Memory usage\n",
+    "\n",
+    "Data volumes are getting larger and larger, and we are getting to the point where a typical dataset is now larger than the amount of system memory available to process the dataset with. In many cases then, we can no longer just load an entire dataset into memory and perform data processing on the dataset.\n",
```

Comment:
I feel that users should be exposed to the notion of `out-of-core processing` as a concept along with deferred data and lazy evaluation. Can that be blended in here? Using the terms without over explaining them

Reply:
Yup, good point 👍